doc/ref_model/chapters/chapter03.md

```diff
@@ -287,5 +287,20 @@ The terms Container Infrastructure Service Instance and Container Infrastructure
 <a name="3.8.2"></a>
 ### 3.8.2 Levels of accelerator consumption
 
-<a name="3.8.3"></a>
-### 3.8.3 Decoupling applications from infrastructure with Hardware Acceleration
+<a name="3.8.9"></a>
+### 3.8.9 Decoupling Applications from Infrastructure and Platform with Hardware Acceleration
+
+[Decoupling](https://github.com/cntt-n/CNTT/blob/master/doc/common/glossary.md#cloud-platform-abstraction-related-terminology) applications from hardware accelerator is normally accomplished using drivers that, if available, are preferred with standardised interfaces across vendors and their products, or if not available then through drivers specific to the vendor hardware device. Decoupling infrastructure software using standardised interfaces from hardware accelerator vendors is also preferred, and, if not available, is comparatively less of an issue by using one or few software infrastructure functions.
```

Comment:
Decoupling software from vendors sounds a bit strange. Maybe remove "from hardware accelerator vendors". I guess the main point is the "standardised interfaces". 
End of the sentence is unclear.

Reply:
> 
> 
> Decoupling software from vendors sounds a bit strange. Maybe remove "from hardware accelerator vendors". I guess the main point is the "standardised interfaces".
> End of the sentence is unclear.

Yes. Now deleted "from hardware accelerator vendors" and reworded the second part.